### DISCUSSION TITLE
기여 질문

### DISCUSSION THREAD

**Discussion by ghost** [hsl0/mediawiki-skin-float#3] [Q&A]
border radius나 그림자 깊이 같은 걸 조금 수정하고 싶은데 어떻게 기여하면 될까요? 수정 전에 논의하거나 해야할 사항이 있나요?

**Comment by hsl0** [comment]
issues에서 논의를 하고 올려도 되고 바로 PR을 올려서 변경사항의 요지를 잘 설명하셔도 됩니다. 하지만 수정하는 이유를 설명하지 않고 PR을 올리면 거부되거나 승인되지 않을 것입니다.

**Reply by ghost** [reply]
네 그냥 리퀘를 올리면 안될 것 같아 문의드렸습니다

**Reply by hsl0** [reply]
기여는 과감히 하시면 됩니다. :) 설득만 잘 해주세요
